.buildkite/scripts/dra_release.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common.sh
+
+VERSION="$(awk '/const DefaultVersion/{print $NF}' version/version.go | tr -d '"')"
+FOLDER_PATH="build/distributions"
+BASE_DIR="${WORKSPACE}/${FOLDER_PATH}"
+DRA_OUTPUT="release-manager.out"
+PROJECT="fleet-server"
+TYPE=${1}
+BRANCH="${BUILDKITE_BRANCH}"
+
+if [[ "${VERSION}" == *"-SNAPSHOT"* || "${VERSION}" == "" ]]; then
+    echo "The 'version' parameter is required and it cannot contain the suffix '-SNAPSHOT'."
+    exit 1
+fi
+
+if [[ "${PROJECT}" == "" ]]; then
+    echo "The 'project' parameter is required."
+    exit 1
+fi
```

Comment:
PROJECT is already set as variable in line 11 in this script. Should it be moved as a env as part of the step definition in the pipeline yaml?

Why is it needed this safeguard? Is this script intended to be copied into other repositories ?

Reply:
It was implemented in the original pipeline with hard-coded values. And It looks like a double-checking because it's an important stage.